api.py

```diff
@@ -124,44 +100,17 @@ def commit(self):
         '''
         Saves user data to the file.
         '''
-        data = {
-            'users': {},
-            'whispers': {}
```

Comment:
can you change it back so that this will get saved as 
```json
{
    "whispers": {
        // whispers
    }
}
```
instead of
```json
{
    // whispers
}
```

Reply:
god dammit

Reply:
changed